### DISCUSSION TITLE
Default Language

### DISCUSSION THREAD

**Discussion by havvanur160401046** [serenity-is/Serenity#5521] [Help]
I want to make the default language Turkish I tried several ways but it didn't work. How can I do it?

**Comment by choss21** [comment]
Hi

Change File: Initialization\UserCultureProvider.cs example for es-MX

![image](https://user-images.githubusercontent.com/13734677/108865679-17956b80-75b9-11eb-9649-ebdf98442a74.png)

**Reply by havvanur160401046** [reply]
it worked. Thank you:)
